apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -75,26 +78,22 @@ const PaymentsSumInfo: React.FC<IPaymentsSumInfoProps> = ({
                 {...{ type }}
                 strong={true}
             >
-                {getMoneyRender(intl, currencyCode)(message)}
+                {loading ? '...' : getMoneyRender(intl, currencyCode)(message)}
```

Comment:
Don't we need a loader here?

Reply:
It is small field with payments sum (and repeates 3 times). I don't think we need loading animation here